src/main/kotlin/com/toasttab/pulseman/entities/TabValuesV2.kt

```diff
@@ -30,6 +38,6 @@ data class TabValuesV2(
     val authJsonParameters: String?,
     val propertyMap: String?,
     val serializationFormat: SerializationFormat?,
-    val protobufSettings: ProtobufTabValues?,
-    val textSettings: TextTabValues?
+    val protobufSettings: ProtobufTabValuesV2?,
+    val textSettings: TextTabValuesV2?
```

Comment:
I've renamed the old versions to V2 to make it more obvious that every version of the save format has the same subclass versions